### PR TITLE
Create manifest creation script.

### PR DESCRIPTION
**Pull request by morriscb**
Add function, ported from Jupyter notebook to create manifests given a set of data.

### CODE REVIEW

File: src/manifest_builder/manifest_builder.py
Comment (danielsf) [inline src/manifest_builder/manifest_builder.py:13]: These global variables do not appear to be used

**Review by danielsf — CHANGES_REQUESTED** [review]
Just some cosmetic changes

File: src/manifest_builder/manifest_builder.py
Comment (morriscb) [reply src/manifest_builder/manifest_builder.py:162]: Changed to dataset_lookup and dataset.

File: src/manifest_builder/manifest_builder.py
Comment (morriscb) [reply src/manifest_builder/manifest_builder.py:87]: Changed to populate_directories

File: src/manifest_builder/manifest_builder.py
Comment (morriscb) [reply src/manifest_builder/manifest_builder.py:238]: Nice catch.

File: src/manifest_builder/manifest_builder.py
Comment (danielsf) [inline src/manifest_builder/manifest_builder.py:285]: Should the example in the docstring be `SEAAD` (two As), since that is what we call the actual dataset?

**Review by danielsf — APPROVED** [review]
Just one comment on a docstring. This is good to merge, though.

File: src/manifest_builder/manifest_builder.py
Comment (morriscb) [reply src/manifest_builder/manifest_builder.py:285]: No. Pretty sure there are also directories with SEA-AD and we also want to skip those.

**Comment by morriscb** [comment]
Running a check of the manifest (downloading and verifying hashes). Once that's done I'll merge the changes unless there is anything else you'd like addressed.

**Pull request merged**
